### DISCUSSION TITLE
fluxCp item database

### DISCUSSION THREAD

**Discussion by Baker887** [rathena/FluxCP#400] [Q&A]
I installed fluxcp on my site the mob db works correctly but the item db still doesn't work even though ItemInfo.lua is well implemented in the database. Here are some screenshots: 
![site](https://github.com/user-attachments/assets/dd0c2094-54a9-4bc5-b740-717aae57b0c2)

![site2](https://github.com/user-attachments/assets/a15de535-ef95-48f0-b2bc-cf37cae51ded)

here is index.php in modules/item :

![site3](https://github.com/user-attachments/assets/d680c0b9-35fe-48db-9a77-a64e81e4eda8)

here is view.php in modules/item : 

![site4](https://github.com/user-attachments/assets/f0ca9ada-da30-40fd-8668-517dd8b23103)

I try to implement item_db_compat , item_db_re_compat , item_db2_compat , item_db2_re_compat but impossible here is my error message : 

![site5](https://github.com/user-attachments/assets/af16a9f8-e246-4e54-b5a2-ff95a2eaed77)

![image](https://github.com/user-attachments/assets/09affa5e-3810-44db-996d-09c13cfc6db0)

![image](https://github.com/user-attachments/assets/9cc22a6c-3296-466f-a746-486ca8893bf1)

![image](https://github.com/user-attachments/assets/0e6ecda1-fa36-4323-baa1-5b8a5e9b265b)

after spending a few hours on it I don't understand Thank you for your precious help.  <3<3

**Comment by Akkarinage** [comment]
You need to run the item sql files from rAthena first.

**Comment by Baker887** [comment]
ok I did what you said!

![image](https://github.com/user-attachments/assets/a5c3097b-03c6-4d20-b20e-d9206d6da823)

![image](https://github.com/user-attachments/assets/e30d141f-45c9-4736-a25a-f4b7545d796e)

![image](https://github.com/user-attachments/assets/c7ac0142-9981-4779-94d0-0946d0558b48)


Still nothing here! 

I did modify the modules/item/index.php script 

![image](https://github.com/user-attachments/assets/18001cfe-7631-43de-940d-1714f3317754)

and this one modules/item/view.php 

![image](https://github.com/user-attachments/assets/a66b8b0d-7026-47a7-a328-f9578665a5c7)

still nothing in my database I do not understand. Thank you for your help.

**Comment by Akkarinage** [comment]
itemInfo.lua has nothing to do with the items being displayed from your database, it only adds the item desc to the cp_itemdesc table.

What mode are you using; renewal or pre-renewal? Have you made sure the compatibility view files were ran correctly without errors?

You don't need to keep posting screenshots of default code.

**Comment by Baker887** [comment]
![image](https://github.com/user-attachments/assets/3cc993b1-be9f-495c-9bf6-7c7ab5ad32f6)

**Reply by Akkarinage** [reply]
I think the problem is fairly obvious.

**Comment by Baker887** [comment]
can you explain why this error is appearing? I don't understand!
[2024-09-15 18:02:56] [SQLSTATE=42S22] Err 1054: Unknown column 'subtype' in 'field list'

Thank you in advance for your support ....

**Comment by Baker887** [comment]
![image](https://github.com/user-attachments/assets/85d4ffd8-2ace-46c9-88ba-7605c959c226)

problem solved thanks anyway
